<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="UTF-8">
  
  <title>Herramientas software | IMPaCT-Data. Infraestructura de Medicina de Precisión asociada a la Ciencia y la Tecnología</title>
  <meta name="description" content=''>

  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <meta name=”robots” content=”noindex”>

  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  
  
    
      <link rel="alternate" hreflang="es" href="../../../outcomes/software/" title="Spanish">
    
  
  <meta http-equiv="content-language" content="en">

  
  <link rel="icon" sizes="32x32" type="image/png" href="../../../images/icons/favicon.png">
  <link rel="apple-touch-icon" sizes="180x180" type="image/png" href="../../../images/icons/apple-touch-icon.png">

  
  <link href="../../../vendor/jpages/css/jPages.css" rel="stylesheet">

  
  <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
  <link rel="stylesheet" href="../../../style.css">

</head>
<body>
<div id="preloader">
  <div class="preload-content">
    <div id="dento-load"></div>
  </div>
</div>

<header class="header-area">
  
  
  

  
  <div class="main-header-area">
    <div class="classy-nav-container breakpoint-off">
      <div class="container">
        
        <nav class="classy-navbar justify-content-between" id="dentoNav">

          
          <a class="nav-brand" href="../../../index.html"><img class="logo" src="../../../images/logos/impact_data_logo_pink_horitzontal.png" alt=""></a>

          
          <div class="classy-navbar-toggler">
            <span class="navbarToggler"><span></span><span></span><span></span></span>
          </div>

          
          <div class="classy-menu">

            
            <div class="classycloseIcon">
              <div class="cross-wrap"><span class="top"></span><span class="bottom"></span></div>
            </div>

            
<div class="classynav">
  <ul id="nav">
    <li><a href="../../../">Home</a></li>
    <li><a href="../../../#about">About us</a>
      <ul class="dropdown">
        <li><a href="../../../about/impact-data">IMPaCT-Data</a></li>
        <li><a href="../../../about/partners">Partnership</a></li>
        <li><a href="../../../about/roadmap">Roadmap</a></li>
        <li><a href="../../../about/workplan">Workplan</a></li>
        <li><a href="../../../about/impact">IMPaCT</a></li>
        <li><a href="../../../about/isciii_calls">ISCIII Calls</a></li>
        <li><a href="../../../about/joboffers">Job offers</a></li>
      </ul>
    </li>
    <li><a href="../../../outcomes">Outcomes</a>
      <ul class="dropdown">
        <li><a href="../../../outcomes/cloud">Cloud IMPaCT-Data</a></li>
        <li><a href="../../../outcomes/software">Software Tools</a></li>
        
        
        <li><a href="../../../outcomes/guidelines">Guidelines</a></li>
        <li><a href="../../../outcomes/deliverables">Deliverables</a></li>
        <li><a href="../../../outcomes/papers">Papers</a></li>
      </ul>
    </li>
    <li><a href="../../../news">News and Events</a></li>
    <li><a href="mailto:impact-data.coord@bsc.es"><strong><i class="fa fa-envelope"></i></strong></a></li>
    <li><a href="https://twitter.com/hashtag/IMPaCTData?src=hashtag_click&f=live" target="_blank"><strong><i class="fa fa-twitter"></i></strong></a></li>
    
    
  </ul>
</div>


          </div>

          
        </nav>
      </div>
    </div>
  </div>
</header>


<div class="breadcumb-area bg-img" style="background-image: url(/images/bg/impact_banner.png);">
  <div class="container h-100">
    <div class="row h-100 align-items-center">
      <div class="col-12">
        <h2 class="title">Herramientas software</h2>
      </div>
    </div>
  </div>
</div>
<div class="breadcumb--con">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <nav aria-label="breadcrumb">
          <ol class="breadcrumb" id="breadcrumbs">
    <li class="breadcrumb-item"><a href="../../../"><i class="fa fa-home"></i> Home</a></li>
                <li class="breadcrumb-item"><a href="../../../en"></a></li>
                <li class="breadcrumb-item"><a href="../../../en/outcomes"></a></li><li class="breadcrumb-item active"><a href="../../../en/outcomes/software">Herramientas software</a></li></ol>

        </nav>
      </div>
    </div>
  </div>
</div>





<main id="main">

  <section class="inner-page mb-5">
    <div class="container content-container" data-aos="fade-up">
      <div class="row mb-5">
  <div class="col-12 d-table">
    <div class="d-table-cell">
      Las herramientas software para el análisis de datos de IMPaCT-Data se pueden encontrar en el 
      <a  href="https://bio.tools/t?domain=impact-data" target="_blank" style="color:orange">
        <strong>dominio IMPaCT-Data</strong>   
      </a>
      de acceso público en bio.tools.
      <a  href="https://bio.tools/" target="_blank" style="color:orange">
        <strong>Bio.tools</strong>   
      </a>
      es un registro de componentes software y bases de datos dirigida a investigadores en el campo de las ciencias biológicas 
      y biomédicas para facilitarles el trabajo de encontrar, entender, utilizar y citar recursos de uso diario.
     </div>
     <div class="d-table-cell">
       <a href="https://bio.tools/" target="_blank">
            <img src="../../../images/logos/biotools-logo.png" alt="bio.tools" style="max-width: 75px; max-height: 75px;">
        </a>
     </div>
  </div>
</div>









  
  
  

  
  
  

  
  
  

  
  
  

  
  
  

  
  
  


<div id="itemContainer">

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/hipathia" target="_blank">hipathia</a></h4>
    <p class="text-justify">Hipathia is a method for the computation of signal transduction along signaling pathways from transcriptomic data. The method is based on an iterative algorithm which is able to compute the signal intensity passing through the nodes of a network by taking into account the level of expression of each gene and the intensity of the signal arriving to it. It also provides a new approach to functional analysis allowing to compute the signal arriving to the functions annotated to each pathway.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/rd-connect_platform" target="_blank">RD-Connect Genome-Phenome Analysis Platform (GPAP)</a></h4>
    <p class="text-justify">An online tool for diagnosis and gene discovery in rare disease research. The platform features allow identifying disease-causing mutations in rare disease patients and linking them with detailed clinical information.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/disgenet" target="_blank">DisGeNET</a></h4>
    <p class="text-justify">DisGeNET is a discovery platform containing one of the largest publicly available collections of genes and variants associated to human diseases. DisGeNET integrates data from expert curated repositories, GWAS catalogues, animal models and the scientific literature. DisGeNET data are homogeneously annotated with controlled vocabularies and community-driven ontologies. Additionally, several original metrics are provided to assist the prioritization of genotype–phenotype relationships.

The current version of DisGeNET (v7.0) contains 1,134,942 gene-disease associations (GDAs), between 21,671 genes and 30,170 diseases, disorders, traits, and clinical or abnormal human phenotypes, and 369,554 variant-disease associations (VDAs), between 194,515 variants and 14,155 diseases, traits, and phenotypes.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/dismed" target="_blank">DiSMed</a></h4>
    <p class="text-justify">DiSMed is a de-identification methodology for Spanish medical texts based on Named Entity Recognition (NER). It is based on spaCy and partially based on the networks designed by Gillaume Genthial implemented on Tensorflow 1. DiSMed includes both the Python code and the curated dataset, available under request under a research use agreement.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/genecodis" target="_blank">GeneCodis</a></h4>
    <p class="text-justify">GeneCodis is a web-based tool for the ontological analysis of lists of genes, proteins, and regulatory elements like miRNAs, transcription factors, and CpGs.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/mcsea" target="_blank">mCSEA</a></h4>
    <p class="text-justify">Identification of diferentially methylated regions (DMRs) in predefined regions (promoters, CpG islands...) from the human genome using Illumina&#39;s 450K or EPIC microarray data. Provides methods to rank CpG probes based on linear models and includes plotting functions.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/ImaGEO" target="_blank">ImaGEO</a></h4>
    <p class="text-justify">ImaGEO is a web tool for gene expression Meta-Analysis that implements a complete and comprehensive meta-analysis workflow starting from Gene Expression Omnibus (GEO) dataset identifiers. The application integrates GEO datasets, applies different meta-analysis techniques and provides functional analysis results in an easy-to-use environment. ImaGEO is a powerful and useful resource that allows researchers to integrate and perform meta-analysis of GEO datasets to lead robust findings for biomarker discovery studies.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/metagenyo" target="_blank">MetaGenyo</a></h4>
    <p class="text-justify">MetaGenyo is a simple, ready-to-use software which has been designed to perform meta-analysis of genetic association studies.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/metaphors" target="_blank">MetaPhors</a></h4>
    <p class="text-justify">MetaPhOrs is a public repository of phylogeny-based orthologs and paralogs that were computed using phylogenetic trees available in twelve public repositories. Currently, over 117,131,162 of unique homologs are deposited in MetaPhOrs database. These predictions were retrieved from 8,246,911 Maximum Likelihood trees for 4,094 species. For each prediction, MetaPhOrs provides a Consistency Score and Evidence Level describing its goodness, together with number of trees and links to their source databases.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/beyondcell" target="_blank">Beyondcell</a></h4>
    <p class="text-justify">Beyondcell is a computational methodology for identifying tumour cell subpopulations with distinct drug responses in single-cell RNA-seq data and proposing cancer-specific treatments.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/dreimt" target="_blank">DREIMT</a></h4>
    <p class="text-justify">DREIMT is a bioinformatics tool for hypothesis generation and prioritization of drugs capable of modulating immune cell activity from transcriptomics data.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/vulcanSpot" target="_blank">vulcanSpot</a></h4>
    <p class="text-justify">Tool to prioritize therapeutic vulnerabilities in cancer.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/appris" target="_blank">APPRIS</a></h4>
    <p class="text-justify">Annotates variants with biological data such as protein structural information, functionally important residues, conservation of functional domains and evidence of cross-species conservation.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/pandrugs" target="_blank">PanDrugs</a></h4>
    <p class="text-justify">PanDrugs is a method to prioritize anticancer drug treatments according to individual genomic data. PanDrugs current version integrates data from 24 primary sources and supports 56297 drug-target associations obtained from 4804 genes and 9092 unique compounds.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/firedb" target="_blank">FireDB</a></h4>
    <p class="text-justify">A curated inventory of catalytic and biologically relevant small ligand-binding sites.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/trifid" target="_blank">TRIFID</a></h4>
    <p class="text-justify">TRIFID is an ML-based tool trained on the evidence of large-scale proteomics analysis and evolutionary, structural, annotation, splicing, and RNA-seq based features to classify the biologically important splice isoforms.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/fair4health_data_privacy_tool" target="_blank">FAIR4Health Data Privacy Tool</a></h4>
    <p class="text-justify">This is a standalone, desktop application developed by the FAIR4Health project (https://www.fair4health.eu/). The tool aims to handle the privacy challenges exposed by the sensitive health data. It is designed to work on an HL7 FHIR API so that it can be used on top of any standard FHIR Repository as a data de-identification, anonymization, and related actions toolset. The tool accesses FHIR resources, presents metadata to the user, guide the user about the configuration to be applied and then output the processed FHIR resources.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/fair4health_data_curation_tool" target="_blank">FAIR4Health Data Curation Tool</a></h4>
    <p class="text-justify">This is a standalone, desktop application developed by the FAIR4Health project (https://www.fair4health.eu/). The tool is used to connect the health data sources which can be in various formats (Excel files, CSV files, SQL databases) and migrate data into a HL7 FHIR Repository. The tool shows the available FHIR profiles to the user so that he/she can perform mappings appropriately. The tool can also contact a Terminology Server (which is actually another HL7 FHIR Repository) so that data fields can be annotated if coding schemes such as ICD10 or SNOMED-CT are in use.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/apid" target="_blank">APID</a></h4>
    <p class="text-justify">APID Interactomes provides a comprehensive collection of protein interactomes for more than 500 organisms based on the integration of known experimentally validated protein-protein physical interactions (PPIs). Construction of the interactomes is done with a methodological approach to report quality levels and coverage over the proteomes for each organism included.
APID unifies PPIs from primary databases of molecular interactions (BIND, BioGRID, DIP, HPRD, IntAct, MINT) and from experimentally resolved 3D structures (PDB) where more than two distinct proteins have been identified. APID also includes a data visualization web-tool that allows the construction of sub-interactomes using query lists of proteins of interest and the visual exploration of the corresponding networks, including an interactive selection of the properties of the interactions reliability of the &#34;edges&#34;) and a mapping of the functional environment of the proteins (functional annotations of the &#34;nodes&#34;).</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/nanoclust" target="_blank">NanoCLUST</a></h4>
    <p class="text-justify">A species-level analysis of 16S rRNA nanopore sequencing data based on de novo clustering and consensus building.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/iongap" target="_blank">IonGAP</a></h4>
    <p class="text-justify">Publicly available integrated pipeline designed for the assembly and subsequent analysis of Ion Torrent bacterial sequence data. Both its components and their configuration are based on a research process aimed to discover the optimal combination of tools for obtaining good results from single-end reads generated by the Ion Torrent PGM sequencer.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/nanortax" target="_blank">NanoRtax</a></h4>
    <p class="text-justify">NanoRTax is a taxonomic and diversity analysis pipeline built originally for Nanopore 16S rRNA data with real-time analysis support in mind. It combines state-of-the-art classifiers such as Kraken2, Centrifuge and BLAST with downstream analysis steps to provide a framework for the analysis of in-progress sequencing runs. NanoRTax retrieves the final output files in the same structure/format for every classifier which enables more comprehensive tool/database comparison and better benchmarking capabilities. Additionally, NanoRTax includes a web application (./viz_webapp/) for visualizing complete or partial pipeline outputs.

The NanoRTax pipeline is built using Nextflow, a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with conda environments and docker containers making installation trivial and results highly reproducible.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/xicra" target="_blank">XICRA</a></h4>
    <p class="text-justify">Small RNAseq pipeline for paired-end reads</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/plasmidid" target="_blank">PlasmidID</a></h4>
    <p class="text-justify">PlasmidID is a mapping-based, assembly-assisted plasmid identification tool that analyzes and gives graphic solution for plasmid identification.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/iskylims" target="_blank">iSkyLIMS</a></h4>
    <p class="text-justify">Open-source LIMS (laboratory Information Management System) for Next Generation Sequencing sample management, statistics and reports, and bioinformatics analysis service management.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/taranis" target="_blank">Taranis</a></h4>
    <p class="text-justify">cg/wgMLST allele calling software, schema evaluation and allele distance estimation for outbreak reserch.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/tfta" target="_blank">TFTA</a></h4>
    <p class="text-justify">Transcription Factor Target Enrichment Analysis</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/priorr" target="_blank">PriorR</a></h4>
    <p class="text-justify">Priorr is a prioritization program of disease-linked genetic variants devoloped within the Genetics&amp;Genomics Department of La Fundacion Jimenez Diaz University Hospital. Priorr is conceived to analyse the output of the FJD-pipeline of SNVs or CNVs. This software program offers a number of useful functionalities for variant analysis such as: filtering by a virtual panel of genes. manual control of different population frequencies or pathogenicity predictors or filtering out variants that have been already found by another protocol.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/fjd-pipeline" target="_blank">FJD-pipeline</a></h4>
    <p class="text-justify">Pipeline for Single Nucleotide Variants (SNVs) and Copy Number Variation (CNVs) variant calling</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/ptmcode" target="_blank">PTMCode</a></h4>
    <p class="text-justify">PTMCode is a resource of known and predicted functional associations between protein post-translational modifications (PTMs) within and between interacting proteins. It currently contains 316,546 modified sites from 69 different PTM types which are also propagated through ortholgs between 19 different eukaryotic species. A total of 1.6 million sites and 17 million functional associations more than 100,000 proteins can currently be explored.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/mini-isoqlr" target="_blank">Mini-IsoQLR</a></h4>
    <p class="text-justify">This pipeline was developed to detect and quantify isoforms from the expression of minigenes, whose cDNA was sequenced using Oxford Nanopore Technologies (ONT).</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/glowgenes" target="_blank">GLOWgenes</a></h4>
    <p class="text-justify">Prioritization of gene diseases candidates by disease-aware evaluation of heterogeneous evidence networks</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/automatic_segmentation_tool" target="_blank">Automatic segmentation tool</a></h4>
    <p class="text-justify">Tool that automatically detects the histological type of a tumor region of interest from Whole Slide Imaging technique</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/slides_viewer" target="_blank">Slides Viewer</a></h4>
    <p class="text-justify">Tool to visualize WSI and navigate through their zones at different zoom levels.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/linkehr" target="_blank">LinkEHR</a></h4>
    <p class="text-justify">LinkEHR is a set of tools that enables the semantic interoperability of your data by:

Creating clinical information models (archetypes)
Transforming clinical data into standards such as openEHR, HL7 CDA, or ISO 13606</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/liferay" target="_blank">Liferay</a></h4>
    <p class="text-justify">Liferay, Inc., is an open-source company that provides free documentation and paid professional service to users of its software.  Mainly focused on enterprise portal technology, the company has its headquarters in Diamond Bar, California, United States.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/hipathia-gemomics" target="_blank">Hipathia-gemomics</a></h4>
    <p class="text-justify">Using mechanistic models for the clinical interpretation of complex genomic variation.

The sustained generation of genomic data in the last decade has increased the knowledge on the causal mutations of a large number of diseases, especially for highly penetrant Mendelian diseases, typically caused by a unique or a few genes. However, the discovery of causal genes in complex diseases has been far less successful. Many complex diseases are actually a consequence of the failure of complex biological modules, composed by interrelated proteins, which can happen in many different ways, which conferring a multigenic nature to the condition that can hardly be attributed to one or a few genes.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/metabolizer" target="_blank">Metabolizer</a></h4>
    <p class="text-justify">Metabolizer is a web tool for analysis of modular architecture of metabolic pathways using transcriptomic data. Metabolizer calculates impact of modules on production of metabolites. These modules are conserved part of metabolism which starts with substrate(s) and ends with a product.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/cypathia" target="_blank">CyPathia</a></h4>
    <p class="text-justify">CyPathia is a cytoscape app, that provides a user friendly and straightforward interface. The CyPathia app is based on Hipathia Bioconductor package, allowing the Cytoscape community the possibility of using mechanistic models.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/cov-hipathia" target="_blank">CoV-Hipathia</a></h4>
    <p class="text-justify">A web tool implements a mechanistic model of human signaling for the interpretation of the consequences of the combined changes of gene expression levels and/or genomic mutations in the context of signalling pathways known to be involved in the infection by SARS-CoV-2, which are updated with the curated versions released by the COVID-19 Disease Map curation project .</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/impusars" target="_blank">impuSARS</a></h4>
    <p class="text-justify">impuSARS allows the imputation of viral whole genome sequences from partially sequenced samples. Additionally, impuSARS provides the lineage associated to the imputed sequence. impuSARS have been validated with a reference of SARS-CoV-2 sequences.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/spacnacs" target="_blank">SPACNACS</a></h4>
    <p class="text-justify">SPACNACS is a crowdsourcing initiative to provide information about Copy Number Variations of the Spanish population to the scientific/medical community.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/nf-core-viralrecon" target="_blank">nf-core-viralrecon</a></h4>
    <p class="text-justify">nfcore/viralrecon is a bioinformatics analysis pipeline used to perform assembly and intra-host/low-frequency variant calling for viral samples. The pipeline supports short-read Illumina sequencing data from both shotgun (e.g. sequencing directly from clinical samples) and enrichment-based library preparation methods (e.g. amplicon-based: ARTIC SARS-CoV-2 enrichment protocol; or probe-capture-based).</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/phylomedb" target="_blank">PhylomeDB</a></h4>
    <p class="text-justify">PhylomeDB is a public database for complete catalogs of gene phylogenie. It allows users to interactively explore the evolutionary history of genes through the visualization of phylogenetic trees and multiple sequence alignments. Moreover, phylomeDB provides genome-wide orthology and paralogy predictions which are based on the analysis of the phylogenetic trees. The automated pipeline used to reconstruct trees aims at providing a high-quality phylogenetic analysis of different genomes, including Maximum Likelihood tree inference, alignment trimming and evolutionary model testing.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/EvolClust" target="_blank">EvolClust</a></h4>
    <p class="text-justify">EvolClust predicts groups of genes that are conserved in terms of gene order across different species distinguishing it from the background gene order conservation found between species. We define a cluster as a group of homologous proteins that are found grouped together in at least two different genomes and which are more conserved than what is expected for the pair of genomes. The order of the genes inside the cluster is not necessarily conserved. Pairwise clusters are grouped into multi-species families.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/openebench" target="_blank">OpenEBench</a></h4>
    <p class="text-justify">OpenEBench (https://openebench.bsc.es) is the ELIXIR benchmarking and technical monitoring platform for bioinformatics tools, web servers and workflows. OpenEBench is part of the ELIXIR Tools platform and its development is led by the Barcelona Supercomputing Center (BSC) in collaboration with partners within ELIXIR and beyond. OpenEBench, holds a specific infrastructure to monitor software quality. In an initial analysis phase BSC has put together a series of quality metrics taken from a number of sources. The source of such metrics includes documents by the Software Sustainability Institute, recommendations for open source software development, or for software quality metrics. For each metric, a specific source of information have been chosen and the necessary interface implemented.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/mignon" target="_blank">MIGNON</a></h4>
    <p class="text-justify">MIGNON (Mechanistic InteGrative aNalysis Of rNa-seq data) is a versatile workflow to integrate RNA-seq genomic and transcriptomic data into mechanistic models of signaling pathways.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/exphuntersuite" target="_blank">ExpHunterSuite</a></h4>
    <p class="text-justify">ExpHunterSuite is an R package for the comprehensive analysis of transcriptomic data.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/metafun" target="_blank">MetaFun</a></h4>
    <p class="text-justify">MetaFun is a web tool for the integration and functional characterization by unveiling sex differences in multiple omics studies through comprehensive functional meta-analysis.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/domfun" target="_blank">DomFun</a></h4>
    <p class="text-justify">DomFun is a system to assign functions to unknown proteins using a systemic approach without considering their sequence but their domains associated with functional systems. It uses associations calculated between protein domains and functional annotations as training dataset and performs predictions over proteins (using UniProt identifiers) by finding their domains and if they have been associated with functional annotations (in GO molecular functions, biological processes, KEGG and Reactome pathway terms).</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/smaca" target="_blank">SMAca</a></h4>
    <p class="text-justify">SMAca: SMA Carrier Analysis tool. SMN1 copy-number and sequence variant analysis from next generation sequencing data.
SMAca is a python tool to detect putative SMA carriers and estimate the absolute SMN1 copy-number in a population. Moreover, SMAca takes advantage of the knowledge of certain variants specific to SMN1 duplication to also identify the so-called “silent carriers” (i.e. individuals with two copies of SMN1 on one chromosome, but none on the other).</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/csvs" target="_blank">CSVS</a></h4>
    <p class="text-justify">Collaborative Spanish Variant Server (CSVS) is a crowdsourcing initiative to provide information about the genomic variability of the Spanish population to the scientific/medical community. It is useful for filtering polymorphisms and local variations in the process of prioritizing candidate disease genes. Submissions from WES or WGS are accepted.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/NanoDJ" target="_blank">NanoDJ</a></h4>
    <p class="text-justify">Dockerized Jupyter notebook for interactive Oxford Nanopore MinION sequence manipulation and genome assembly.</p>
  </div>

  <div class="box-shadow p-3 mb-3">
    <h4><a href="https://bio.tools/ngscat" target="_blank">ngsCAT</a></h4>
    <p class="text-justify">Tool to assess the efficiency of targeted enrichment sequencing.</p>
  </div>

</div>

<div class="holder mb-5">
</div>

<script type="text/javascript">
  window.onload = function() {
    $("div.holder").jPages({
      containerID: "itemContainer",
      perPage: 5
    });
  }
</script>


    </div>
  </section>

</main>


<footer class="footer-area bg-img bg-gradient-overlay-footer" style="background: linear-gradient(to bottom right, #2e76b5,  #2e76b5, #32c896);">
  <div class="container">
    <div class="row">
      
      <div id='contact' class="col-sm-12 col-md-6 col-lg-4"><div class="single-footer-widget">
  
  <div class="footer-contact">
    <a href="../../../" class="d-block mb-4">
       <img src="../../../images/logos/impact_data_logo_white_300x185.png" class="footer-logo" alt="IMPaCT-Data">
    </a>
    <h5 class="widget-title">Contact with us</h5>
    <div class="line"></div>
    <p>
      <a href="mailto:impact-data.coord@bsc.es"><i class="icon_mail_alt"></i><i>impact-data.coord@bsc.es</i></a>
    </p>
    
    
  </div>
</div>
</div>

      
      <div class="col-sm-12 col-md-6 col-lg-3"><div class="single-footer-widget">
  
  <h5 class="widget-title">Useful links</h5>
  <div class="line"></div>

  
  <nav>
    <ul class="quick-links">
      <li><a href="../../../about/impact-data"><i class="bx bx-chevron-right"></i>IMPaCT-Data</a></li>
      <li><a href="../../../outcomes"><i class="bx bx-chevron-right"></i>Outcomes</a></li>
      <li><a href="../../../news"><i class="bx bx-chevron-right"></i>News and Events</a></li>
      <li><a href="../../../about/isciii_calls"><i class="bx bx-chevron-right"></i>ISCIII Calls</a></li>
      <li><a href="../../../about/joboffers"><i class="bx bx-chevron-right"></i>Job offers</a></li>
    </ul>
  </nav>
</div>
</div>

      
      <div class="col-sm-12 col-md-6 col-lg-5"><div class="single-footer-widget">

    <p class="d-flex">
        <img class="footer-logo ml-auto"
             alt="ISCIII-FEDER"
             title=" Instituto de Salud Carlos III, co-financiado por el Fondo Europeo de Desarrollo Regional "
             src="../../../images/logos/ISCIII_FEDER_lema_vertical.png"
             style="max-width: 300px; max-height: 100px">
    </p>

    <p class="small text-justify">The IMPaCT-Data project (Exp. IMP/00019) has been financed by the Institute of Health Carlos III, co-financed by the European Regional Development Fund (ERDF, “A way to make Europe”).</p>
</div>
</div>
    </div>
  </div>

  
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="copywrite-content">
          <p>
            
            Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="fa fa-heart-o" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank">Colorlib</a>.
            
            Some icons has been created by <a href="https://www.flaticon.es/iconos-gratis/adn" title="icons">Freepik - Flaticon</a>
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>

<a href="#" class="back-to-top float-right"><i class="icofont-simple-up"></i></a>


<script src="../../../js/jquery.min.js"></script>

<script src="../../../js/popper.min.js"></script>

<script src="../../../js/bootstrap.min.js"></script>

<script src="../../../vendor/jpages/js/jPages.js"></script>

<script src="../../../js/dento.bundle.js"></script>

<script src="../../../js/default-assets/active.js"></script>

<script src="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/owl.carousel.min.js"></script>

<script src="https://unpkg.com/isotope-layout@3/dist/isotope.pkgd.min.js"></script>

<script src="../../../js/table-sort.js"></script>

<script src="../../../js/custom.js"></script>
</body>
</html>
